lib/services/ios-project-service.ts

```diff
@@ -20,6 +20,7 @@ class IOSProjectService implements  IPlatformProjectService {
 		private $iOSEmulatorServices: Mobile.IEmulatorPlatformServices) { }
 
 	public get platformData(): IPlatformData {
+		var projectName = options["debug-brk"] ? this.$projectData.projectName + "WithInspector" : this.$projectData.projectName;
```

Comment:
This code is duplicated on 3 places. I suggest you to expose here `projectName` and use it from other places.